ISSUE_TEMPLATE/testing.md

```diff
@@ -92,7 +92,32 @@ fedora-coreos-stream-generator -releases=https://fcos-builds.s3.amazonaws.com/pr
 - [ ] Once approved, merge it and push it to S3:
 
 ```
-aws s3 cp --acl=public-read streams/testing.json s3://fcos-builds/streams/testing.json --cache-control max-age=60
+aws s3 cp --acl public-read --cache-control 'max-age=60' streams/testing.json s3://fcos-builds/streams/testing.json
 ```
 
 - [ ] Verify the new version shows up on [the download page](https://getfedora.org/en/coreos/download/)
+
+## Rollout updates to existing hosts (updates metadata)
+
+From a checkout of this repo:
+
+- [ ] Edit `updates/testing.json`, replacing the oldest rollout with your new one:
+  - [ ] Set "version" field to the new version
+  - [ ] Set "start\_epoch" field to a future timestamp for the rollout start (e.g. `date +%s <DATE>`)
+  - [ ] Set "start\_percentage" field to `0.0`
+  - [ ] Set "duration\_minutes" field to a reasonable rollout window (e.g. `2880` for 48h)
+- [ ] Edit `updates/testing.json` "last-modified" field to current time (e.g. `date -u -Is`)
```

Comment:
should this just be a sub-bullet in the previous list since it's another `edit updates/testing.json` action? or maybe you mean `streams/testing.json` instead?

Reply:
Yes, I can indent further.